globalvalues.py

```diff
@@ -28,8 +28,10 @@
 DEFAULT_TCP_PORT = 5100
 DEFAULT_SENDER_MODE = 'udp'
 DEFAULT_DATALOG = '/home/pi/data-log.txt'
+DEFAULT_DATA_BACKLOG_FILE = '/home/pi/data_backlog_file.txt'
+DEFAULT_PROTOCOL = 'old'
 
-DEFAULT_INTERVAL_NORMAL = 300
+DEFAULT_INTERVAL_NORMAL = 30
```

Comment:
Don't forget to revert this...
In general for testing and development, it's better to test with `sudo python manager.py -i 30` for example, so you don't have to change default values and potentially risk forgetting to set them back.